tests/metrics_test.go

```diff
@@ -67,4 +67,10 @@ var _ = Describe("Metrics", func() {
 			expectAppLabelsRestoreAfterUpdate(&prometheusRuleRes)
 		})
 	})
+
+	Context("alerts", func() {
+		It("[test_id:CNV-7851]should have available runbook URLs", func() {
```

Comment:
@erkanerol should the test number have the "CNV-" prefix?

Reply:
I missed that :( That id should be polarion id of that test case. If there is no such a polarion test case, you should create it or delete that test_id label from there.

Reply:
Here is the polarion test case: https://polarion.engineering.redhat.com/polarion/#/project/CNV/workitem?id=CNV-7851
Should I change the id to 7851?

Reply:
Yes. Please drop `CNV-` prefix.

Reply:
Thanks, done